### ISSUE TITLE
Version 8 beta suggested improvements

### ISSUE THREAD

**Issue by justin808** [shakacode/react_on_rails#833]
1. Need to document all values in this file or slim down 
https://github.com/shakacode/react_on_rails/blob/master/lib/generators/react_on_rails/templates/base/base/config/webpack/paths.yml
```yaml
default: &default
  output: public
  assets: webpack
  manifest: manifest.json
  node_modules: client/node_modules
  source: client/app

development:
  <<: *default
  assets: webpack/development

test:
  <<: *default
  assets: webpack/test

production:
  <<: *default
  assets: webpack/production
```

**Comment by justin808** [comment]
@kaizencodes any ideas?

**Comment by kaizencodes** [comment]
We definitely need assets. Manifestplugin uses it.

**Comment by justin808** [comment]
@kaizencodes look at the code above. The Rails env overrides the default. So by removing assets, I mean removing the default one only.

**Comment by justin808** [comment]
We need to document how this file is the glue between our configuration of the manifest plugin and the output of webpack with the new Rails asset helpers.

**Comment by justin808** [comment]
Fixed in 8.0.
